lib/adapters/sequelize.js

```diff
@@ -41,24 +41,124 @@ function translateOptions(spell, options) {
   if (offset) spell.$offset(offset);
 }
 
+const setScopeToSpell = (scope) => (spell) => {
```

Comment:
I smell redux middleware

Reply:
😂 Because [Spell accept function scope](https://github.com/cyjake/leoric/blob/1fff0b95243db4ba11f766f67c7dffece742a201/lib/drivers/abstract/spellbook.js#L692)